EIPS/eip-3525.md

```diff
@@ -12,7 +12,7 @@ requires: 165, 721
 
 ## Simple Summary
 
-This is a standard for semi-fungible tokens. The set of smart contract interfaces described in this document defines an ERC-721 extension/compatible token standard. This standard introduces an <ID, SLOT, VALUE> triple scalar model that represents the semi-fungible structure of a token. It also introduces new transfer models as well as approval models that reflect the semi-fungible nature of the tokens.
+This is a standard for semi-fungible tokens. The set of smart contract interfaces described in this document defines an ERC-721 compatible token standard. This standard introduces an <ID, SLOT, VALUE> triple scalar model that represents the semi-fungible structure of a token. It also introduces new transfer models as well as approval models that reflect the semi-fungible nature of the tokens.
 
 ## Abstract
 
```

Comment:
Making comments here, as I cannot suggest edits to un-modified lines:

- Throughout: Currently, EIPs with the ERC category are still referred to as EIPs. I would hold off on this though, as #5190 is in the works.
- L19: A relative link to EIP-721 needs to be added.
- L35-L39: No external links.
- L47-L48: No need to link to the EIPs here
- L49-L50: No external links.
- Throughout: Refer to EIP-3525 as "This EIP"